KeyboardTeacher/Key.pde

```diff
@@ -0,0 +1,48 @@
+class Key {
+  PFont textFont;
+
+  int x, y, keyWidth, keyHeight, textSize, transparency;
+  String text;
+  color strokeColor, fillColor, textColor;
+
+  Key(int X, int Y, int W, int H, String T) {
+    x = X;
+    y = Y;
+    keyWidth = W;
+    keyHeight = H;
+    text = T;
+    textSize = text.length() / keyWidth;
+  }
+
+  void keyColor(color s, color f, color t) {
+    strokeColor = s;
+    fillColor = f;
+    textColor = t;
+  }
+
+  void dinamicColorChange(color a1, color a2, color a3, color b1, color b2, color b3, color c1, color c2, color c3, int tra) {
+    transparency = tra;

```

Comment:
If you want to assign a parameter to a variable of the class, a very common pattern is:
```this.transparency = trasparency;```
In this way you do not have to invent new exoteric variable names, and everything is plain and clear.